### PR TITLE
feat(graph): Add params for d3 force

### LINKED ISSUE #3758: Graph2 needs publish property for additional d3 force features

### PR DESCRIPTION
**Pull request by jbrundage**
Fixes #3758

Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [ ] The commit is signed.
- [ ] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

**Comment by jbrundage** [comment]
Before and After this change, but using the same publish properties:
image

**Comment by jbrundage** [comment]
Before and after this change, but setting .forceDirectedMaxDistance(200)
image

**Comment by jbrundage** [comment]
...and with .forceDirectedMinDistance(5) there's only a slight difference (it lessens the force severity when vertices are tightly packed):
image

**Comment by jbrundage** [comment]
...and .forceDirectedForceStrength(0.1) pulls everything toward the middle:
image

**Comment by jbrundage** [comment]
...and here's .forceDirectedForceStrength(0.05):
image

File: packages/graph/package.json [hunk 1]
```diff
@@ -70,7 +70,7 @@
         "cpx": "1.5.0",
         "d3-array": "1.2.4",
         "d3-drag": "1.2.4",
-        "d3-force": "1.2.1",
+        "d3-force": "2.1.1",
```
Comment (GordonSmith) [inline packages/graph/package.json:73]: This change will not work in IE11 (do we care?) - I suspect your changes will work with 1.2.1 as well?

File: packages/graph/src/graph2/graph.ts [hunk 2]
```diff
@@ -1224,6 +1233,9 @@ Graph2.prototype.publish("forceDirectedIterations", 300, "number", "Iterations",
 Graph2.prototype.publish("forceDirectedLinkDistance", 300, "number", "Target distance between linked nodes", null, { disable: (w: Graph2) => w.layout().indexOf("ForceDirected") !== 0 });
 Graph2.prototype.publish("forceDirectedLinkStrength", 1, "number", "Strength (rigidity) of links", null, { disable: (w: Graph2) => w.layout().indexOf("ForceDirected") !== 0 });
 Graph2.prototype.publish("forceDirectedPinCentroid", false, "boolean", "Pin centroid to center", null, { disable: (w: Graph2) => w.layout().indexOf("ForceDirected") !== 0 });
+Graph2.prototype.publish("forceDirectedForceStrength", 0, "number", "Strength of center force", null, { disable: (w: Graph2) => w.layout().indexOf("ForceDirected") !== 0 });
+Graph2.prototype.publish("forceDirectedMinDistance", 1, "number", "Max distance between nodes", null, { disable: (w: Graph2) => w.layout().indexOf("ForceDirected") !== 0 });
```
Comment (GordonSmith) [inline packages/graph/src/graph2/graph.ts:1237]: Description is wrong

**Review by GordonSmith — CHANGES_REQUESTED** [review]

**Review by GordonSmith — APPROVED** [review]

**Pull request merged**